src/vs/workbench/services/actions/common/menusExtensionPoint.ts

```diff
@@ -291,7 +291,11 @@ const apiMenus: IAPIMenu[] = [
 		description: localize('webview.context', "The webview context menu"),
 		proposed: 'contribWebviewContext'
 	},
-
+	{
+		key: 'mergeEditor/result/context',
```

Comment:
Needs proposal, see L292, and needs a correct name. This isn't a context menu but a title-menu, so something like `mergeEditor/result/title`. The fact that it goes into the dropdown/overlfow is the concept of groups